### ISSUE TITLE
[cssom] Add resolved value special case for background-image and similar?

### ISSUE THREAD

**Issue by andruud** [w3c/csswg-drafts#9659]
As far as I can tell, the [resolved value](https://drafts.csswg.org/cssom/#resolved-value) of `background-image` is defined by specs as the _computed value_. This means that e.g. `currentcolor` within gradients should _not_ be resolved in the return value of `getComputedStyle(e).backgroundImage`, yet all browsers appear to do this anyway.

Should we change cssom and say that the resolved value for `background-image` is the _used_ value? (May also be relevant for other properties, e.g. `border-image`). Or do we try to fix implementations instead? @emilio

**Comment by emilio** [comment]
It's unclear we want the "used" value, what does that imply for the length of the lists etc? It's just "computed with colors resolved", right?

**Comment by andruud** [comment]
Yeah, I wasn't sure either. Do you think we should spec resolved="computed with colors resolved", then?

**Comment by emilio** [comment]
I think that's fine, yeah I think we don't expose currentColor in the computed style anywhere.
